tools/semgrep/ci/fmt-printf.yml

```diff
@@ -0,0 +1,11 @@
+rules:
+  - id: fmt.Printf
+    languages: [go]
+    message: fmt.Printf is forbidden outside of cmd and test files
+    pattern: fmt.Printf
```

Comment:
Might as well throw in a `fmt.Println` check too!

Reply:
:+1: